torch/onnx/symbolic_opset11.py

```diff
@@ -874,7 +874,26 @@ def _get_arange_dtype(dtype):
         dtype = symbolic_helper._maybe_get_const(dtype, "i")
         return dtype
 
-    if len(args) == 2 or len(args) == 5:
+    if len(args) == 2 and all(map(lambda val: isinstance(val, int), args)):
+        # aten::arange(Scalar start, Scalar end)
+        dtype = torch.int64
+        # Start index.
+        start = g.op(
```

Comment:
They are values from FX graph; if not expected, it's `torch.fx.symbolic_trace`'s problem and we will fix there. On the other hand, how do you create constant tensor?

Reply:
> how do you create constant tensor?

Roughly like `g.op("Constant", value_t=torch.tensor(range(start, end, step))`. But that's fine, feel free to ignore this part of comment.

> They are values from FX graph; if not expected, it's torch.fx.symbolic_trace's problem and we will fix there.

The expectation of dynamic vs static needs to be very clear on both parts (frontend and symbolic). "Silently converted as constant" has been a major pain point for current exporter. IMO we should do it right from the start. i.e. we should only permit writing symbolic functions like this, if `int` type in input IR definition is guaranteed to just mean static `int`, instead of carrying the ambiguity of possibly being dynamic (this is what happens in torchscript IR). I think we are already doing better with ATenLib.

This is not PR blocking, some thoughts as we move forward.